collector/src/test/java/com/bitmovin/analytics/error/ErrorDataTest.kt

```diff
@@ -0,0 +1,23 @@
+package com.bitmovin.analytics.error
+
+import com.bitmovin.analytics.data.ErrorData
+import com.bitmovin.analytics.utils.DataSerializer
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class UtilTest {
```

Comment:
```suggestion
class ErrorDataTest {
```
This is the same file as `collector-exoplayer/src/test/java/com/bitmovin/analytics/error/ErrorDataTest.kt` except that the className is different.
Is it necessary to duplicate those?

Reply:
Nope, was a leftover. thx for pointing out